servicebox-app/src/main/resources/templates/claims/list.html

```diff
@@ -26,13 +45,15 @@
             </tbody>
         </table>
     </div>
-    <footer class="footer">
+    <footer class="container footer">
```

Comment:
Is this expected?

Reply:
Yes. We need the class container otherwise the different `div` `header`, `body` and `footer` are not aligned. I will check if we can declare it only within a parent `div`

Reply:
See 360e56b